samri/pipelines/preprocess.py

```diff
@@ -476,10 +484,33 @@ def generic(bids_base, template,
 				(s_warp, datasink, [('output_image', 'anat')]),
 				])
 
-		workflow_connections.extend([
-			(get_f_scan, get_s_scan, [('subject_session', 'selector')]),
-			(get_s_scan, s_warp, [('nii_name','output_image')]),
-			(get_s_scan, s_biascorrect, [('nii_path', 'input_image')]),
+		if model_prediction_mask == True:
```

Comment:
Why not just use `if model_prediction_mask:` ?

Reply:
yes that would be cleaner

```suggestion
		if model_prediction_mask:
```